config/Betterloadingscreen/tips/en_US.txt

```diff
@@ -1,66 +1,150 @@
-#Maximal characters for a tip: 77
-#The line below, including the "#" are 77 chars
-#Adventurer's backpack? Did you mean integrated crafting grid, bed and fluid
+#Tips for the loading screen
+#
+#Thanks to 0lafe, bedrill, boubou19, chochem, interestingusernam3, glowredman, GTNH-Afx237v7, leagris, Lewissaber, Nvidiot, Regian24, richardhendricks, yukieiji and others for tip suggestions
+#
+#Maximal character count for a tip: roughly 80 (more can go out of the screen)
+#
 Infinite water sources are disabled
-Got a question? Join our Discord server
+Got a question? Join our Discord server: discord.com/invite/EXshrPV
 Don't give ideas to 0lafe
-Don't feed your machines after midnight
+Don't feed your machines after midnight!
 Make sure you have installed a backup mod
 Material tiers play a role when breaking pipes
 If a machine catches fire, it can explode
-Adding water to an empty but hot Boiler will cause an explosion
-Avoid eldritch obelisks
+Adding water to an empty but hot Boiler will make a loud bang!
+Avoid eldritch obelisks!
 You can bind the quests menu to a key, instead of using the book
 Pam's gardens can be picked up with right-click
 Placing a garden makes it spread
-Water garden can grow on land
+Water gardens can grow on land
 Battlegear slots are convenient for holding weapons
 Taking lava without gloves hurts!
 Watch out, food loses saturation
 Loot Games give helpful rewards
 Using too many translocators can cause TPS lag
-Be sure to check out what you can do with mouse tweaks
-Protect your machines from rain
-Build multiblocks within the same chunk
+Be sure to check out what you can do with Mouse Tweaks
+Protect your machines from rain!
+Build multiblocks within the same chunk!
 You will lose your first piece of aluminium dust in the EBF
-Shift-right click with a wrench makes a fluid pipe input-only
-The bending machine makes plates more efficiently
-Some multiblocks can share walls
-You can not use the front side of machines
-Disable a machine with a soft mallet if it can not finish a recipe
-Forestry worktables are a must!
-Try the midnight theme for the quests menu
-Try the realistic sky resourcepack
-Literally flint and steel
-Tinker's tools can levelup
-Farm Glowflowers for glowstone
-Making steel armour? Check out the composite chestplate
-Adventurer's backpack: integrated crafting grid, bed and fluid storage
-Beware of cable power loss
-Machines that get a higher voltage than they can handle explode
+Shift-right-clicking with a wrench on a fluid pipe makes it input-only
+The Bending Machine makes plates more efficiently
+Multiblocks can share walls
+You can't use the front side of machines
+Disable a machine with a Soft Mallet if it can not finish a recipe
```

Comment:
change "can not" to "can't"

Reply:
I would prefer "can not" since "can't" is colloquial.

Reply:
"cannot" would be better if staying with less colloquial

Reply:
Tips of this kind tend to be colloquial. Think of the splash messages in minecraft

Reply:
Can't makes most sense in this context.